webrtc.html

```diff
@@ -4624,10 +4624,12 @@ <h2>Methods</h2>
         </div>
         <p>For the purposes of this API, the <a>[[\Certificate]]</a> slot
         contains unstructured binary data.</p>
-        <p>Note that an <code>RTCCertificate</code> might not directly hold
-        private keying material, this might be stored in a secure module.</p>
         <p>The <code>RTCCertificate</code> object can be stored and retrieved
-        from persistent storage by an application. When a <a>user agent</a> is
+        from persistent storage by an application. Note that an
+        <code>RTCCertificate</code> might not directly hold
+        private keying material, this might be stored in a secure module,
+        in which case the <a>[[\KeyingMaterial]]</a> internal slot might not
+        be storable and retrievable. When a <a>user agent</a> is
```

Comment:
If we store it and retrieve it on the same host, would we assume that the KSM can still be accessed?

Reply:
all the web crpto seem pretty vague about this. I think the answer is "maybe". AFAIC this is all way under specified. So for example, if the user and told the browser to clear the locally storage, it would almost certainly be gone - I think it is often tied to saving in a one of the web databases / local storage options. So if that is gone, it is probably gone even if it is still in the KSM. How memory on the KSM is ever reclaimed is not clear to me. But .... I don't think we can fix this in WebRTC spec.